src/database/DataTypes/Requisition.js

```diff
@@ -47,6 +51,10 @@ export class Requisition extends Realm.Object {
     this.daysToSupply = months * 30;
   }
 
+  get isLinkedToRequisition() {
+    return typeof this.linkedRequisition === 'object';
```

Comment:
Good point, I am pretty sure realm will store 'undefined' in object key that has no value. I will change this check to keep with consistency.